tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/env python3
+
+# generate_tls13_compat_tests.py
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Generate TLSv1.3 Compat test cases
+
+"""
+
+import sys
+import os
+import abc
+import argparse
+import itertools
+
+# pylint: disable=useless-super-delegation
+
+CERTIFICATES = {
+    'ecdsa_secp256r1_sha256': (
+        'data_files/ecdsa_secp256r1.crt',
+        'data_files/ecdsa_secp256r1.key'),
+    'ecdsa_secp384r1_sha384': (
+        'data_files/ecdsa_secp384r1.crt',
+        'data_files/ecdsa_secp384r1.key'),
+    'ecdsa_secp521r1_sha512': (
+        'data_files/ecdsa_secp521r1.crt',
+        'data_files/ecdsa_secp521r1.key'),
+    'rsa_pss_rsae_sha256': (
+        'data_files/server2-sha256.crt', 'data_files/server2.key'
+    )
+}
+
+
+CAFILE = {
```

Comment:
Why not merging CERTIFICATES and CAFILE to have only one dictionary indexed by the signature algorithms ?